src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -174,7 +174,18 @@ protected function compileTableExpression($sql, $constraint, $query)
             return "select * from ({$sql}) where rownum {$constraint}";
         }
 
```

Comment:
Adding the codes seems to fix 904 issues from migration to actual app execution:

```suggestion
        $orders = $query->orders;

        foreach($orders as $key => $order) {
            if (isset($order['sql'])) {
                [$column, $direction] = explode(' ', $order['sql']);

                if (Str::contains($column, '.')) {
                    [$table, $column] = explode('.', $column);
                    $orders[$key]['sql'] = '"T1".'.$column.' '.$direction;
                }

                continue;
            }

            if (isset($order['column']) && Str::contains($order['column'], '.')) {
                [,$column] = explode('.', $order['column']);
                $orders[$key]['column'] = 't1.'.$column;
            }
        }
```